Tests/XCUITests/NavigationTest.swift

```diff
@@ -152,19 +152,6 @@ class NavigationTest: BaseTestCase {
         XCTAssertTrue(app.buttons["Bookmark Link"].exists, "The option is not shown")
     }
 
-    func testLongPressLinkOptionsPrivateMode() {
```

Comment:
Is this test covered elsewhere?

Reply:
It has been moved to the PrivateBrowsingTest suite